static/docs/get-started/import-data.md

```diff
@@ -0,0 +1,70 @@
+# Import Data
+
+We've seen how to [push](/doc/get-started/store-data) and
+[pull](/doc/get-started/retrieve-data) data a <abbr>DVC project</abbr>'s
+[remote](/doc/commands-reference/remote). But what if we wanted to integrate a
+dataset or ML model produced in one project into another project? (As an
+[external dependency](/doc/user-guide/external-dependencies))
```

Comment:
`))` ->`).)` (end of sentence)

Reply:
Hmmm it's strange indeed but wasn't a full sentence so no period `.`. I'm moving the question mark `?` to the end instead (: Thanks